Packs/Cryptosim/IncidentFields/incidentfield-Cryptosim_Event_Start_Date.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
@haldunerbay Why not use the existing "Start Time" field?

Reply:
Same reason like category id. To specify event start using event word.

Reply:
Again, preferably, you would use the existing field If you find this as an important addition, you would need to add the prefix and associate it only to the relevant incident type.

Reply:
I added prefixes and all of them associated with my incident type 'Cryptosim Correlation Alert'.